_format/1.0/index.md

```diff
@@ -577,6 +577,7 @@ by clients and servers, and they **MUST** meet all of the following conditions:
 - Member names **MUST** contain only the allowed characters listed below.
 - Member names **MUST** start and end with a "globally allowed character",
   as defined below.
+- Member names **SHOULD** use camelCasing   
```

Comment:
is there a reason for SHOULD instead of MUST in this case?

Reply:
fair point. I guess I wanted to leave it open just in case. But maybe we should just require it.

Reply:
I think a *MUST* is more clear about our goal.